src/Dialog/formComponents/fields/StringField.js

```diff
@@ -79,7 +80,10 @@ function StringField(props) {
     const Widget = getWidget(schema, widget, widgets);
     return <Widget {...commonProps} placeholder={placeholder} />;
   }
-  return <TextWidget {...commonProps} placeholder={placeholder} />;
+  return <TextWidget {...commonProps}
+    placeholder={placeholder}
+    step={step}
```

Comment:
Please move `step` property to `commonProps`, because nice to have this props in other widgets.